app/src/main/res/layout/notification.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ImageView
+            android:id="@+id/app_logo"
+            android:layout_marginLeft="10dp"
+            android:layout_marginTop="10dp"
+            android:layout_height="70dp"
+            android:layout_width="70dp"
+            android:src="@drawable/show_fridge_notification" />
+
+        <TextView
+            android:id="@+id/title_logo"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_toRightOf="@id/app_logo"
+            android:layout_marginLeft="10dp"
+            android:layout_marginTop="15dp"
+            android:text="Title"
+            android:textSize="20sp"
+            android:textStyle="bold"/>
+        <TextView
+            android:id="@+id/message"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/title_logo"
+            android:layout_toRightOf="@id/app_logo"
+            android:layout_marginLeft="10dp"
+            android:text="the text description"
```

Comment:
Can be removed I think

Reply:
no, the whole notification script is needed, there is the connection in getRemoteView. To create the notification